### DISCUSSION TITLE
horizontal scrollable datepicker

### DISCUSSION THREAD

**Discussion by trogalko** [dotnet/maui#22180] [Ideas]
### Description

Please add the horizontally scrollable datepicker to the .NET 8 MAUI and Blazor, that would be lovely. We could scroll to the left until reach the minimum date, dan to the right up to the maximum date. Here is example of visual appearance :
![image](https://github.com/dotnet/maui/assets/107082/a4f6789e-00fc-4c2d-b582-4e8d92a3737d)

### Public API Changes

```csharp var scrollableDatePicker = new ScrollableDatePicker();```
```scrollableDatePicker.Orientation = Orientation.Horizontal;```

### Intended Use-Case

Horizontal scrollable datepicker would take less space than reguler datepicker and offer much easier experience for users to use.

**Comment by JohnHDev** [comment]
That looks pretty easy to implement, you could try writing it yourself and submitting as a PR?
